apps/api-documenter/src/cli/GenerateAction.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+
+import { ApiDocumenterCommandLine } from './ApiDocumenterCommandLine';
+import { BaseAction } from './BaseAction';
+
+import { ExperimentYamlDocumenter } from '../documenters/ExperimentYamlDocumenter';
+import { ApiModel } from '@microsoft/api-extractor-model';
+import { DocumenterConfig } from '../documenters/DocumenterConfig';
+import { FileSystem } from '@microsoft/node-core-library';
+import { IConfigFile } from '../documenters/IConfigFile';
```

Comment:
Can you coalesce these and maybe group them by internal/external paths?

Reply:
Sorted 👍